.github/workflows/e2e-test-ci-v2-cron-dev.yml

```diff
@@ -26,19 +26,14 @@ on:
     branches:
       - master
       - v1.6.0
-  pull_request:
-    branches:
-      - master
-      - v1.6.0
-    types: [opened, synchronize, reopened, review_requested]
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-dev
+  group: ${{ github.workflow }}-dev
   cancel-in-progress: true
 
 jobs:
   changes:
-    if: ${{ (!github.event.pull_request.draft || github.event.schedule == '0 0 * * *') }}
+    if: ${{ github.event.schedule == '0 0 * * *') }}
```

Comment:
we can just remove this condition.

Reply:
Can I refer to any file?
Is this line used to run ci at regular intervals?

Reply:
As the trigger condition has already been specified through the "on" clause at the beginning of the file, this judgment is unnecessary.

Reply:
Yeah, There is an update!